### ISSUE TITLE
Keep calendar view state after reopening from DatePicker or in another way.

### ISSUE THREAD

**Issue by hanastasov** [IgniteUI/igniteui-angular#6373]
* igniteui-angular version: 9.1.0

## Steps to reproduce  

1. Go to https://stackblitz.com/edit/igniteui-angular-issue-6373
2. Open the calendar from the date picker.
3. Select a value in the second visible month
4. Reopen the calendar.

## Result  
The month that was second in the view is now first in the view.

## Expected result  
Keep the order of the months.

**Comment by ddincheva** [comment]
Hello @zdrawku  and @hanastasov ,
Before starting to implement this feature I performed some additional research and from what I can see the current behavior is expected and similar to all competitors I manage to find. 
![datePicker](https://user-images.githubusercontent.com/34240583/104283708-9e82fe80-54b9-11eb-8970-8583eb0b9d26.gif)

In addition to that, the datepicker does not have a property viewDate /sets the date that is presented in the first data view /. Also in the Calenddar component if the viewDate is not set and there is a selected date, the selected date value is used for viewDate/ issue #8173/;
In my opinion, this behavior is quite similar to the following:
![datePickerSS](https://user-images.githubusercontent.com/34240583/104283702-9a56e100-54b9-11eb-839a-4730bd09f11e.gif)

Please let me know what do you think and should I proceed with the implementation of this feature.

**Comment by hanastasov** [comment]
As I understand, the issue comes down to a `viewDate` property in the date picker, which is an other question at all, and it seems that the concept of a date p9icker does not need a `viewDate`.

Regarding that a selected date is used as a `viewDate` in the calendar - I reopened issue #8173 , because I think it introduced a wrong behavior.

**Comment by hanastasov** [comment]
Looking closer to the gif from Delta, I see that the gif shows that only a Depart date is selected, so next time you open the calendar, the depart date is in the first month, which is convenient for the user. Once you select a Return date in the second month view, close the date picker. Opening it again will keep the views as it was.

AirBnb, booking and wizzair date pickers behave in the same way. Will add a gif shortly.

**Comment by hanastasov** [comment]
Nevertheless, the igxDatePicker does not support date ranges, so we cannot campore it to Delta, Airbnb, Wizzair, etc, so I think we can consider the issue out of scope for the DatePicker.
